pom.xml

```diff
@@ -1364,6 +1364,38 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>oss</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-gpg-plugin</artifactId>
+            <version>1.5</version>
+            <executions>
+              <execution>
+                <id>sign-artifacts</id>
+                <goals>
+                  <goal>sign</goal>
+                </goals>
+                <phase>verify</phase>
```

Comment:
Is this more a deploy phase goal? I suppose it probably has to be done before the deploy phase?

Reply:
I think technically that is fine. But all the examples show it as being done at the verify stage. 
https://maven.apache.org/plugins/maven-gpg-plugin/usage.html

Maybe because the deployer is "verifying" that the code is signed and authorized 🤷